test/e2e/run

```diff
@@ -29,17 +29,18 @@ DESCRIPTION = "Execute a single Litani run and test the resulting run.json"
 EPILOG = "See test/e2e/README for the organization of this test suite"
 
 
-def run_cmd(cmd):
+def run_cmd(cmd, check):
```

Comment:
This function `run_cmd` is only called from one place---`run_litani` (line 64 below). Since that call gives an explicit argument to `check`, I don't see a need for this parameter to be optional. If I start calling this function from other places and the argument becomes cumbersome, I'd remove it at that point